scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -25,7 +25,7 @@
   },
```

Comment:
<p>Do we need the FRAME_RATE here, and can we also remove TRACK_BUFFER?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>